a draft 'vision' (#26)
pharmaR/regulatory-r-repo-wg: README.md

# Regulatory R Package Repository

> Using R in submissions to healthcare regulators is challenging since the 
> quality of any packages used must be ensured and evidence of ensuring this
> quality must be made available on request. 

Reviewer: I tried to more explicitly highlight the "why" and the desired outcomes.

```suggestion
Submissions to healthcare regulators rely on quality software for reliable, 
reproducible analysis, with evidence of this quality must be available on 
request.
To support recent trends in regulated industries toward adoption of open 
tooling, we aim to adapt these quality deliverables to the new ways that 
software is developed and distributed. It is our goal to embed these 
activities into shared resources for package distribution to make quality
more consistent and transparent.
```

Author: hmmm, this sounds a bit like we want to dev packages ourselves, isn't that rather the scope of the ASA WG on SWE https://rconsortium.github.io/asa-biop-swe-wg/ ?

Reviewer: Hm - yeah, rereading it again I think there's still room for improvement. I'm not seeing the package development angle that you're talking about, but it definitely doesn't strike me as clear. Maybe we leave this out for now.

This working group strives to identify and prototype at least one technical

> framework that can support a transparent, open, dynamic, cross-industry approach

Reviewer 2: ```suggestion
framework that can support a transparent, open, dynamic, cross-industry approach
```

> of establishing and maintaining a 'repository' of core R packages with

Reviewer: Just stepping back the "core" qualifier, since I think that scope is still TBD

```suggestion
of establishing and maintaining a 'repository' of R packages with
```

> accompanying evidence of their quality that can be used to simplify necessary

Author: "[...] evidence of their quality and the assessment criteria, [...]" ?

> accompanying evidence of their quality that can be used to simplify necessary

Reviewer 3: Yes, that sounds better!

> accompanying evidence of their quality that can be used to simplify necessary

Author: done

in-house validation processes as much as possible.
Such a cross-industry 'repository' could help with burden-sharing of
validation efforts, improve quality via transparent, open peer review, and

> de-risk the use of important R packages for regulatory submissions.

Reviewer: Generalizing the "important" thing just to reflect that this could pertain to all public packages.

"important" can be a bit ambiguous - is it important in how many reverse dependencies it supports? important for regulatory reporting? important due to frequency of use? I think it's fine to keep it open-ended for now.

```suggestion
de-risk the use of public R packages for regulatory submissions.
```
